tests/locust/locustfile.py

```diff
@@ -0,0 +1,171 @@
+        self.client.get('/search?page=1&size=20&type=Glossary&sort=title')
+
+    @task
+    def search_experiment_cms(self):
+        self.client.get('/search?experiment=CMS')
+
+    @task
+    def search_type_news(self):
+        self.client.get('/search?type=News')
+
+    @task
+    def api_search(self):
+        self.client.get('/api/records')
+
+    @task
+    def api_search_atlas(self):
+        self.client.get('/api/records?page=1&size=20&experiment=ATLAS')
+
+    @task
+    def api_search_alice(self):
+        self.client.get('/api/records?page=1&size=20&experiment=ALICE')
+
+    @task
+    def api_search_cms(self):
+        self.client.get('/api/records?page=1&size=20&experiment=CMS')
+
+    @task
+    def api_search_lhcb(self):
+        self.client.get('/api/records?page=1&size=20&experiment=LHCb')
+
+    @task
+    def api_search_dataset(self):
+        self.client.get('/api/records?page=1&size=20&experiment=Dataset')
+
+    @task
+    def api_search_software(self):
+        self.client.get('/api/records?page=1&size=20&experiment=Software')
+
+    @task
+    def api_search_environment(self):
+        self.client.get('/api/records?page=1&size=20&experiment=Environment')
+
+    @task
+    def api_search_documentation(self):
+        self.client.get('/api/records?page=1&size=20&experiment=Documentation')
+
+    @task
+    def api_search_keywords_education(self):
+        self.client.get(
+            '/api/records?page=1&size=20&experiment=keywords:education')
+
+    @task
+    def api_search_collision_data(self):
+        self.client.get(
+            '/api/records?page=1&size=20&experiment=collision%20data')
+
+    @task
+    def docs_alice_getting_started(self):
+        self.client.get('/docs/alice-getting-started')
+
+    @task
+    def docs_atlas_higgs_machine_learning_challenge(self):
+        self.client.get('/docs/atlas-higgs-machine-learning-challenge')
+
+    @task
+    def docs_cms_guide_for_education(self):
+        self.client.get('/docs/cms-guide-for-education')
+
+    @task
+    def docs_cms_guide_for_research(self):
+        self.client.get('/docs/cms-guide-for-research')
+
+    @task
+    def docs_cms_summer_student_report_2017(self):
+        self.client.get('/docs/cms-summer-student-report-2017')
+
+    @task
+    def docs_cms_the_future_is_open_2017(self):
+        self.client.get('/docs/cms-the-future-is-open-2017')
+
+    @task
+    def docs_lhcb_getting_started(self):
+        self.client.get('/docs/lhcb-getting-started')
+
+    @task
+    def docs_docs_welcome(self):
+        self.client.get('/docs/welcome')
+
+    @task
+    def visualise_events_cms(self):
+        self.client.get('/visualise/events/cms')
+
+    @task
+    def visualise_histograms(self):
+        self.client.get('/visualise/histograms')
+
+    @task
+    def record_detail_1002(self):
+        self.client.get('/record/1002')
+
+    @task
+    def record_detail_5001(self):
+        self.client.get('/record/5001')
```

Comment:
Might be nice to add a record from each different collection and release year.  Here they are:

(1) HTML/Jinja:

```
/record/1
/record/1000
/record/1002
/record/1050
/record/1052
/record/1100
/record/11000
/record/1120
/record/1200
/record/1204
/record/1300
/record/15
/record/1700
/record/1701
/record/1800
/record/1801
/record/1803
/record/200
/record/2000
/record/201
/record/203
/record/220
/record/221
/record/230
/record/233
/record/250
/record/252
/record/253
/record/300
/record/3000
/record/320
/record/328
/record/35
/record/3500
/record/352
/record/3600
/record/3700
/record/3800
/record/3860
/record/3900
/record/3901
/record/40
/record/400
/record/4000
/record/402
/record/41
/record/410
/record/450
/record/451
/record/452
/record/453
/record/460
/record/463
/record/4900
/record/50
/record/5000
/record/5001
/record/5003
/record/5100
/record/5104
/record/545
/record/55
/record/550
/record/5500
/record/551
/record/553
/record/554
/record/556
/record/560
/record/60
/record/600
/record/6000
/record/6026
/record/6100
/record/614
/record/6200
/record/7000
/record/7100
/record/7125
/record/7299
```

(2) REST API

```
1
/api/records/1000
/api/records/1002
/api/records/1050
/api/records/1052
/api/records/1100
/api/records/11000
/api/records/1120
/api/records/1200
/api/records/1204
/api/records/1300
/api/records/15
/api/records/1700
/api/records/1701
/api/records/1800
/api/records/1801
/api/records/1803
/api/records/200
/api/records/2000
/api/records/201
/api/records/203
/api/records/220
/api/records/221
/api/records/230
/api/records/233
/api/records/250
/api/records/252
/api/records/253
/api/records/300
/api/records/3000
/api/records/320
/api/records/328
/api/records/35
/api/records/3500
/api/records/352
/api/records/3600
/api/records/3700
/api/records/3800
/api/records/3860
/api/records/3900
/api/records/3901
/api/records/40
/api/records/400
/api/records/4000
/api/records/402
/api/records/41
/api/records/410
/api/records/450
/api/records/451
/api/records/452
/api/records/453
/api/records/460
/api/records/463
/api/records/4900
/api/records/50
/api/records/5000
/api/records/5001
/api/records/5003
/api/records/5100
/api/records/5104
/api/records/545
/api/records/55
/api/records/550
/api/records/5500
/api/records/551
/api/records/553
/api/records/554
/api/records/556
/api/records/560
/api/records/60
/api/records/600
/api/records/6000
/api/records/6026
/api/records/6100
/api/records/614
/api/records/6200
/api/records/7000
/api/records/7100
/api/records/7125
/api/records/7299
```